src/language/cli/output.ts

```diff
@@ -27,7 +26,10 @@ export const handleOutput = async (
   if (typeof noColorArg !== 'boolean') {
     throw new Error('Unsupported value for --no-color')
   } else if (noColorArg === true) {
-    kleur.enabled = false
+    // Warning: the global state mutation here means that we can't style text in static contexts!
+    // Functions like `node:util`'s `styleText` shouldn't be called from the top level of modules.
+    delete process.env['FORCE_COLOR']
+    process.env['NO_COLOR'] = 'true'
```

Comment:
This feels even worse than the global state mutation I had to do for `kleur`, but on the upside is more general: now `--no-color` will also disable styling in stack traces, `console.log` output, etc.